PhoneNumberKit/UI/TextField.swift

```diff
@@ -59,7 +61,11 @@ open class PhoneNumberTextField: UITextField, UITextFieldDelegate {
         }
     }
 
-    let partialFormatter: PartialFormatter
+    private var _partialFormatter: PartialFormatter?
```

Comment:
what is the reason that `let partialFormatter: PartialFormatter` wasn't sufficient.

even in the new patch, the only time partialFormatter is ever referenced is to be read.

 the only time it's new backing var is set is as part of init, which is a perfectly viable place to set a `let`.

Reply:
1. `let partialFormatter` must be initialized **before** `super.init` call.
2. `self.defaultRegion` cannot be accessed only **after** `super.init` call.

You can try it by yourself.